purchase/advanced/analyze.rst

```diff
@@ -1,99 +1,119 @@
+If your company regularly purchase products, you have several reasons to track and analyze your
+performance. The insights gathered can help you to better forecast and plan your future orders. For
+example, you can assess if your business is dependent on particular vendors, and the data can help
+you negotiate discounts on prices.
 
--  You can see how dependant from a supplier your company is;
+Generate customized reports
+===========================
 
--  you can negotiate discounts on prices;
+Go to *Reporting* to access the purchase analysis. Simply by accessing the reporting dashboard, you
+can get a quick overview of your actual performance. By default, the reporting dashboard displays a
+line chart presenting the untaxed amount of your purchase orders per day, and below it, key metrics
+and a pivot table.
 
--  You can check the average delivery time per supplier;
+.. image:: media/analyze-reporting-dashboard.png
+   :align: center
+   :alt: Reporting dashboard in Odoo Purchase
 
--  Etc.
+While the data initially presented is useful, there are several tools and features you can use to
+get even more insights on your purchases.
 
-For example, an IT products reseller that issues dozens of purchase
-orders to several suppliers each week may want to measure for each
-product the total price paid for each vendor and the delivery delay. The
-insights gathered by the company will help it to better analyze,
-forecast and plan their future orders.
+Use filters to select the data you need
+---------------------------------------
 
-Configuration
-=============
+Odoo offers several default filters you can use and combine by clicking on *Filters*. When you
+select one or several filters, Odoo looks for all the orders that match at least one of the filters
+you have chosen and populates the chart, key metrics, and pivot table with the data. The
+pre-configured filters are:
 
-Install the Purchase Management module
---------------------------------------
+#. All *Requests for Quotation*
 
-From the **Apps** menu, search and install the **Purchase Management**
-module.
+#. All *Purchase Orders*, except canceled ones
 
-.. image:: media/analyze03.png
-    :align: center
+#. *Confirmation Date Last Year* includes all orders that were confirmed the previous year,
+   canceled purchase orders included
 
-Issue some purchase orders
---------------------------
+#. *Order Date* includes all orders - request for quotations and purchases orders (canceled ones
+   included) - depending on their date of creation
 
-Of course, in order to analyze your vendors' performance, you need to
-issue some **Request For Quotations** (RfQ) and confirm some **Purchase
-Orders**.
+#. *Confirmation Date* includes all confirmed orders, canceled ones included, depending on their
+   date of confirmation
 
-Analyzing your vendors
-======================
+.. note:: When you have to select a period, you can use multiple years, and, with at least one year
```

Comment:
As your are changing the whole file anyway, please start the content of your notes and tips on the next line. It's easier to read and maintain.